pyproject.toml

```diff
@@ -98,14 +98,19 @@ exclude = [
     'modal/_vendor',
 ]
 line-length = 120
-lint.ignore = ['E741']
-lint.select = [
+
+[tool.ruff.lint]
+ignore = ['E741', 'F841']
```

Comment:
Not sure why `inv lint` is now catching [F841](https://docs.astral.sh/ruff/rules/unused-variable/) violations. 🤔